include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -256,7 +314,7 @@ class operational_domain_impl
             truth_table{tt},
             params{ps},
             stats{st},
-            output_bdl_pairs{detect_bdl_pairs<Lyt>(layout, sidb_technology::cell_type::OUTPUT, params.bdl_params)},
+            output_bdl_pairs{detect_bdl_pairs<Lyt>(lyt, sidb_technology::cell_type::OUTPUT, ps.bdl_params)},
```

Comment:
Why these changes?

Reply:
Should be equivalent, I'm just curious

Reply:
I don't know why I did this 😅. I will undo this. Thank you!